notebooks/06_grouping_methods_exercises.ipynb

```diff
@@ -696,7 +696,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.10"
+   "version": "3.9.6"
```

Comment:
The version shouldn't change in both notebooks.

Reply:
Updated notebook with the requested changes.